spring-integration-core/src/main/java/org/springframework/integration/store/SimpleMessageStore.java

```diff
@@ -72,23 +78,51 @@
 	public SimpleMessageStore(int individualCapacity, int groupCapacity) {
 		this(individualCapacity, groupCapacity, new DefaultLockRegistry());
 	}
-
+    /**
+     * See {@link #SimpleMessageStore(int, int)}.
+     *
+     * @since @TODO
+     * @param individualCapacity The message capacity.
+     * @param groupCapacity The capacity of each group.
+     * @param maxDelayInMilliseconds The time to wait if the store is at max capacity
+     * 
+     */
+    public SimpleMessageStore(int individualCapacity, int groupCapacity, int maxDelayInMilliseconds) {
```

Comment:
yes, saw this in travis, will leave final variable name to you. I don't want to impose